.coveragerc

```diff
@@ -27,6 +27,9 @@ omit =
 
     homeassistant/components/envisalink.py
     homeassistant/components/*/envisalink.py
+    
+    homeassistant/components/icloud.py
```

Comment:
I wouldn't be able to test everything, only the logic from the events and the idevices, but I wouldn't be able to test the icloud api itself. Is that ok?